.clang-format

```
@@ -8,3 +8,8 @@ BinPackArguments: false
 BinPackParameters: false
 AllowShortFunctionsOnASingleLine: Empty
 IncludeBlocks: Preserve
+
+# Comment pragma for Filecoin System Test Matrix (https://github.com/filecoin-project/system-test-matrix)
+# Comment format is '//stm:tags=...;ignore=...' with no whitespace. This is a go convention for comments that are not
+# meant to be read by humans and must not be changed by clang-format
+CommentPragmas: 'stm'
```

[CommentPragmas is regex](https://clang.llvm.org/docs/ClangFormatStyleOptions.html).  
`'stm'` searches substring, so `//unrelated stm` would match too.
```suggestion
CommentPragmas: '^stm:'
```